src/docker/docker-compose.yml

```
@@ -0,0 +1,16 @@
+version: '3'
+services:
+  keycloak:
+    build: .
+    environment:
+      # Default user name und password
+      - KEYCLOAK_ADMIN=admin
+      - KEYCLOAK_ADMIN_PASSWORD=admin
+      #- KC_LOG_LEVEL=DEBUG
+    command: start-dev
+    volumes:
+      - keycloak-data:/opt/keycloak/data
```

```suggestion
      - keycloak-data:/opt/keycloak/data
      - ../../target/keycloak-ip-authenticator.jar:/opt/keycloak/providers/keycloak-ip-authenticator.jar:
```
Du kannst die Datei direkt in den Cointainer mounten, dann benötigst du kein Dockerfile.

---

../../ ggfs mit ${PWD} anpassen